packages/puppeteer/README.md

```diff
@@ -155,6 +155,22 @@ new AxePuppeteer(page)
   .disableRules('color-contrast');
 ```
 
+### AxePuppeteer#disableFrame(selector: string)
+
+Skips specific frame with selector provided. Accepts a String of a single selector. **Subsequent calls to `AxePuppeteer#options`, `AxePuppeteer#disableFrame` will override specified options.**
+
+```js
+new AxePuppeteer(page).disableFrame('color-contrast');
```

Comment:
may be call the frame `my-frame`? instead of a rule name like `color-contrast`?

Reply:
sorry copy pasta fail